.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -53,15 +53,18 @@ body:
       multiple: true
       options:
         - Danbooru
+        - E621.net (and E926.net)
```

Comment:
I think the YAML changes can be produced in the new PR.

Reply:
Ok. What should I do

Reply:
Remove these changes in this PR, and then open a new PR for them.